sympy/functions/elementary/hyperbolic.py

```diff
@@ -209,6 +209,11 @@ class cosh(HyperbolicFunction):
     sinh, tanh, acosh
     """
 
+    @property
+    def is_positive(self):
+        if self._eval_is_real():
```

Comment:
This should not be based on reality, since cosh also takes negative real values.